src/save.ts

```diff
@@ -57,7 +57,15 @@ async function saveCache() {
       await mc.fPutObject(bucket, object, archivePath, {});
       core.info("Cache saved to s3 successfully");
     } catch (e) {
-      core.info("Save s3 cache failed: " + e.message);
+      core.info("Save s3 cache failed: " + e.toString());
+      core.debug(
+        `Full error: ${{
+          name: e.name,
```

Comment:
hmm. will `e` always have all these properties available? Why not just stringify it, and get rid of the rest?

Reply:
We could do that and/or remove the stringify to avoid leaking secrets (that sometimes happen when you stringify the error of an http request, they add the headers in the error).
There is no guarantee that those props are available since you can throw anything in JS (like a string), but if you throw an error they are generally there. 
This was just a hack I put together to debug failing cache saving, but I thought it might be useful to others (somethings the S3 copy-cat behave a bit differently).

Reply:
Sounds good to me, my question is actually simpler: if we have `stringify(e)` why do we need others? i.e. `name: e.name` etc?

Reply:
We don't, I think I added the stringify after because I was missing props